ios/RNSScreenWindowTraits.mm

```diff
@@ -56,7 +56,9 @@ + (void)updateHomeIndicatorAutoHidden
 #endif
   {
     if (@available(iOS 11.0, *)) {
+#if !TARGET_OS_VISION
       [UIApplication.sharedApplication.keyWindow.rootViewController setNeedsUpdateOfHomeIndicatorAutoHidden];
```

Comment:
Should we use `RCTKeyWindow` instead of `UIApplication.sharedApplication.keyWindow` here?

Reply:
Yes, definitely - I will add it